tests/test_cms_config.py

```diff
@@ -17,6 +24,96 @@
 from djangocms_versioning.test_utils.polls.models import Poll, PollContent
 
 
+req_factory = RequestFactory()
+
+
+class PageContentVersioningBehaviourTestCase(CMSTestCase):
+
+    def setUp(self):
+        self.site = Site.objects.get_current()
+        self.user = self.get_superuser()
+        self.language = 'en'
+        self.title = 'test page'
+
+        self.page = create_page(
```

Comment:
We shouldn't be using the api to create a page. The PageContent and a PageUrl entry should be all that we need:

version = PageContentVersionFactory(content__language='en', state=PUBLISHED,)
PageUrlFactory(page=version.content.page, language='en', path="hello", slug="hello",)

Reply:
Why shouldn't we be using the api to create a page?